pkg/networkmanager/firewall-client.js

```diff
@@ -155,7 +155,18 @@ function getServices() {
                                    'org.fedoraproject.FirewallD1.zone',
                                    'getServices', [z])
                 .then(reply => fetchServiceInfos(reply[0]))
-                .then(services => services.map(s => firewall.enabledServices.add(s.id)));
+                .then(services => {
+                    let promises = [];
+                    for (let s of services) {
+                        firewall.enabledServices.add(s.id);
+                        if (s.includes.length)
+                            promises.push(fetchServiceInfos(s.includes));
+                    }
+                    // We can't use Promise.all() here until cockpit is able to dispatch es2015 promises
+                    // https://github.com/cockpit-project/cockpit/issues/10956
+                    // eslint-disable-next-line cockpit/no-cockpit-all
```

Comment:
This should actually be obsolete now. But let's keep it here, and we can drop all cockpit.all() in a separate PR.